Formula/zile.rb

```diff
@@ -21,6 +21,8 @@ class Zile < Formula
   depends_on "help2man" => :build
   depends_on "pkg-config" => :build
   depends_on "bdw-gc"
+  depends_on "glib"
+  depends_on "libgee"
```

Comment:
Are these build-only? From the `NEWS` file:

```
** Miscellaneous

  Zile has been rewritten in Vala. This makes the code considerably shorter
  (~6kLoC as compared with 8kLoC for the C version) and easier to
  maintain. It now requires glib and libgee to build, but a Vala compiler is
  not required, as the generated C sources are shipped in release tarballs.
```

Reply:
Oops. I did read the `NEWS` file, but I missed that part.

Reply:
honestly ignoring the things that I just read in the instructions is the source of most of my problems...